content/news/2025/2025-08-04-4diac-podcast.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Tune In! Announcing the New 4diac Podcast"
+date: 2025-08-04T00:00:00-00:00
+categories: ["4diacpodcast"]
+type: "newsitem"
+description:  |
+   Get ready to enhance your understanding of industrial automation, IEC 61499, and the world of 4diac – wherever you are! We're thrilled to announce the launch of our brand new podcast, "*The Future of Industrial Control: Decoding IEC 61499.*"
+
+---
+
+
+## Tune In! Announcing the New 4diac Podcast: **The Future of Industrial Control: Decoding IEC 61499**!
```

Comment:
Please do not repeat title and description. our hugo news template does that automatically.